### PR TITLE
EDGECLOUD-3187 Fix performance mode cloudlet map

### PR DESCRIPTION
**Pull request by brucearmstrong**
Garner's update for EDGECLOUD-3184 would have fixed the main problem, but I made this update to get a debug version of the app available immediately. It's simpler, so I'm keeping it.

There's also a fix for the Settings screen.

### CODE REVIEW

File: android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/MainActivity.java [hunk 1]
```diff
@@ -915,14 +916,18 @@ private String getBadgeText(Cloudlet cloudlet) {
...
+        }
         runOnUiThread(new Runnable() {
             @Override
             public void run() {
                 Cloudlet cloudlet = null;
                 for (int i = 0; i < CloudletListHolder.getSingleton().getCloudletList().size(); i++) {
                     cloudlet = CloudletListHolder.getSingleton().getCloudletList().valueAt(i);
-                    if(cloudlet.getMarker().getPosition().latitude == closestCloudlet.getCloudletLocation().getLatitude() &&
-                            cloudlet.getMarker().getPosition().longitude == closestCloudlet.getCloudletLocation().getLongitude() ) {
+                    Log.i(TAG, "Checking: "+closestCloudlet.getFqdn()+" "+cloudlet.getFqdn());
```
Comment (franklin-huang-mobiledgex) [inline android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/MainActivity.java:929]: Where does closestCloudlet come from?

Comment (ah1053) [reply android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/MainActivity.java:929]: MatchingEnginerHelper > findClodulet

Comment (franklin-huang-mobiledgex) [reply android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/MainActivity.java:929]: Ahh gotcha. What does this code do @brucearmstrong ? Check if the findCloudletReply cloudlet is in the appInstListReply list of cloudlets?

Comment (brucearmstrong) [reply android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/MainActivity.java:929]: At startup, I do a getAppInstList to populate the map with cloudlets. If the user does a findCloudlet, I want to highlight the cloudlet icon (turn it green) and draw a line from the mobile icon to the cloudlet icon. That why I'm searching for a match in the existing list.

**Pull request merged**